### DISCUSSION TITLE
Norton VPN blocks progress

### DISCUSSION THREAD

**Discussion by Yiaagaitia** [openaudible/openaudible#1252] [General]
After reconnecting to Audible, it shows as downloading for about 10 seconds then stops and goes back to status of needing to be downloaded.

No error or message.

After shutting off Norton VPN, download worked.

I have found a lot of sites changed access about 11 days ago if you are using Norton VPN.

I don't have another VPN so can not speak to others.

**Comment by openaudible** [comment]
Have not heard that. Not sure if that is audible blocking known vpn's or if the VPN is buggy or overloaded. Good to know.
